### DISCUSSION TITLE
Card shows `could not fetch total commits`

### DISCUSSION THREAD

**Discussion by jonesy-b-dev** [anuraghazra/github-readme-stats#3429] [Q&A] [answered]
### Describe the bug

My starts card with my total commits really often doesn't load and gives always the error named in the title.  as shown in the first image .

### Expected behavior

I expect the card to load normally, I can't upload an image now where it's working since it doesn't work at the moment.

### Screenshots / Live demo link

![image](https://github.com/anuraghazra/github-readme-stats/assets/59418915/6aa41314-ea41-4d8b-a371-57b2fcaad009)

**Comment by qwerty541** [comment, marked as answer]
Hey, @jonesy-b-dev! Our project use GitHub API under the hood to fetch users data. You can see this error when upstream API for some reason respond with error. It cannot be fixed from our side. It should be automatically resolved after sime time. In my experience it does not occures too frequently.

**Reply by jonesy-b-dev** [reply]
Okay good to know! Yeah it doesn't happen that often but I see it happen more often these days. Then it's okay, not your guys fault that it's not working.
